app/views/application_submissions/show.haml

```diff
@@ -1,33 +1,38 @@
 %h1 #{@record.position.name} Application Record
 
-submitted
-= format_date_time @record.created_at
-by:
-= @record.user.name_and_email
+%p submitted #{format_date_time @record.created_at}
+%p by #{@record.user.name_and_email}
 = button_to 'Print this page',
-  application_submission_path(@record, format: :pdf), method: :get
+  application_submission_path(@record, format: :pdf),
+  method: :get,
+  class: 'btn btn-secondary'
 - if @record.rejected?
-  = button_to 'Re-review Applicant',
-    unreject_application_submission_path(@application_submission, id: @record.id)
+  %p
+    = button_to 'Re-review Applicant',
+      unreject_application_submission_path(@application_submission, id: @record.id),
+      class: 'btn btn-primary'
 - if !@record.saved_for_later? && @record.note_for_later.present?
-  %em
-    This application had previously been saved for later review. Your note:
-  = @record.note_for_later
-.data_container
-  .data_row
-    %h2.prompt_item Question
-    %h2.response_item Response
-  - @record.data.each do |prompt, response, data_type, question_id|
-    - if data_type == 'heading'
-      .data_row
-        .heading_item= prompt.upcase
-    - elsif data_type == 'explanation'
-      .data_row
-        .explanation_item= render_markdown prompt
-    - else
-      .data_row
-        .prompt_item= prompt
-        .response_item= response
+  %p
+    %i This application had previously been saved for later review. Your note:
+  %p
+    = @record.note_for_later
+%table.table.table-bordered
```

Comment:
Not sure this should be a table.

At the very least, the width needs to be fixed because the presence of a long "explanation" squishes the answers column:

![2020-01-13_17-30](https://user-images.githubusercontent.com/240275/72297691-6e94e700-362a-11ea-92a7-59f1f339f585.png)

<sub>Questions in green, answers in red</sub>